### ISSUE TITLE
`PLATEAU Technical Report #xx` pattern not working

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-plateau#7]
```sh
$ bundle exec relaton fetch "PLATEAU Technical Report #93" -f yaml
[relaton-plateau] WARN: (PLATEAU Technical Report #93) Not found.
No matching bibliographic entry found
```

**Comment by andrew2net** [comment]
@ronaldtse Technical Report's IDs are like `PLATEAU Tech Report #93`.

**Comment by ronaldtse** [comment]
@andrew2net we need "Technical Report" in addition to "Tech Report" as well. Please fix this.

**Comment by ronaldtse** [comment]
> @ronaldtse Technical Report's IDs are like `PLATEAU Tech Report #93`.

BTW, why was this implemented as `Tech Report #{n}`?

See here:
* https://github.com/relaton/relaton-plateau/issues/3#issuecomment-2230000491

It clearly says `PLATEAU Technical Report #93`.

**Comment by andrew2net** [comment]
> BTW, why was this implemented as `Tech Report #{n}`?

@ronaldtse it was your implementation. I didn't change it. Should we change all the IDs to `Technical Report #{n}`?

**Comment by ronaldtse** [comment]
@andrew2net I see, I wasn't think about the pattern but just wanted you to obtain the information. Yes please, change to "Technical Report #{n}".

**Comment by andrew2net** [comment]
Fixed in v1.19.1
